main.py

```diff
@@ -0,0 +1,55 @@
+import random
+import art
+import os
+from game_data import data
+
+# compare() calculates which choice has more followers and checks if user's answer is that choice. Returns bool
+def compare(choice_a: dict, choice_b: dict) -> bool:
+    user_answer = input('Who has more instagram followers? Type "A" or "B": ').lower()
+    
+    if choice_a['follower_count'] > choice_b['follower_count']:
+        solution = 'a'
+    elif choice_b['follower_count'] > choice_a['follower_count']:
+        solution = 'b'
+    else:
+        solution = user_answer
+        
+    if user_answer == solution:
+        return True
+    else: 
+        return False
+
+# coninue_game() calls compare(). While compare returns True game will continue. Returns final score
+def continue_game(subject1: dict, subject2: dict) -> int:
+    end = compare(choice_a = subject1, choice_b = subject2)
+    score = 0
+    
+    while end:
+        os.system('cls')
+        score += 1
+        print(f"You're right! Current score: {score}.")
+        subject1 = subject2
+        subject2 = random.choice(data)
+        print_UI(subject1 = subject1, subject2 = subject2)
```

Comment:
Don't mix snake_case and capital letters

Reply:
Mental note taken. will only code lowercase snake_case

Reply:
Cool